client/web/compose/src/components/PageBlocks/Navigation/NavTypes/ComposePage.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <tr>
+    <td />
+    <td>
+      <b-form-group :label="$t('navigation.fieldLabel')">
+        <b-form-input
+          v-model="value.options.item.label"
+          type="text"
+          size="sm"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.composePage')">
+        <vue-select
+          key="pageID"
+          v-model="value.options.item.pageID"
+          :placeholder="$t('navigation.none')"
+          :options="tree"
+          class="bg-white"
+          label="title"
+          :reduce="f => f.pageID"
+          option-value="pageID"
+          option-text="title"
+          @input="updateLabelValue"
+        />
+      </b-form-group>
+    </td>
+    <td>
+      <b-form-group :label="$t('navigation.openIn')">
+        <b-form-select
+          v-model="value.options.item.target"
+          :options="openInType"
```

Comment:
openInType makes me think this holds the value of what is selected. 
Try to keep naming convention of appending options to the end of the variable of the options